src/test/groovy/com/github/rodionovsasha/commentservice/services/TopicServiceTest.groovy

```diff
@@ -0,0 +1,181 @@
+package com.github.rodionovsasha.commentservice.services
+
+import com.github.rodionovsasha.commentservice.BaseTest
+import com.github.rodionovsasha.commentservice.exceptions.InactiveUserException
+import com.github.rodionovsasha.commentservice.exceptions.TopicAccessException
+import com.github.rodionovsasha.commentservice.exceptions.TopicNotFoundException
+import com.github.rodionovsasha.commentservice.exceptions.UserNotFoundException
+import com.github.rodionovsasha.commentservice.repositories.TopicRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.data.domain.Sort
+
+class TopicServiceTest extends BaseTest {
+    @Autowired
+    TopicService topicService
+    @Autowired
+    TopicRepository topicRepository
+
+    def "start creates a new topic"() {
+        when:
+        def topic = topicService.start("D'oh!", 1)
+
+        then:
+        topic.id instanceof Long
+        topic.title == "D'oh!"
+        !topic.archived
+        topic.date instanceof Date
+    }
+
+    def "start creates a new topic with owner"() {
+        when:
+        def topic = topicService.start("D'oh!", 1)
+
+        then:
+        topic.owner.id instanceof Long
```

Comment:
I think here we need to check the exact owner it since we set it above: `topicService.start("D'oh!", 1)`. Checking other fields of owner is redundant, imo. So the test could look like:
```groovy
given:
def OWNER_ID = 1 // Homer

when:
def topic = topicService.start("D'oh!", OWNER_ID)

then:
topic.owner.id == OWNER_ID
```